lib/components/viewers/stop-viewer.js

```diff
@@ -189,7 +190,19 @@ class StopViewer extends Component {
         <div className="header-text">
           {stopData ? (
             <>
-              {agencyCount <= 1 && <OperatorLogo operator={stationOperator} />}
+              {agencyCount <= 1 && stationOperator && (
+                <OperatorLogo
+                  alt={intl.formatMessage(
+                    {
+                      id: 'components.StopViewer.operatorLogoAriaLabel'
+                    },
+                    {
+                      operatorName: stationOperator.name
+                    }
+                  )}
+                  operator={stationOperator}
+                />
+              )}
               <h1 style={{ paddingLeft: '0.5ch' }}>{stopData.name}</h1>
             </>
```

Comment:
Could we add the OperatorLogo into the h1? Otherwise you get the name of the operator before the header which feels a little strange: 

![image](https://user-images.githubusercontent.com/115499534/225366714-f0b3c83d-9764-409b-be9e-9fb426105c54.png)